astropy/coordinates/builtin_frames/cirs_observed_transforms.py

```diff
@@ -52,7 +61,7 @@ def cirs_to_altaz(cirs_coo, altaz_frame):
                          altaz_frame.relative_humidity,
                          altaz_frame.obswl.value)
 
-    az, zen, ha, obs_dec, obs_ra = erfa.atioq(cirs_ra, cirs_dec, astrom)
+    az, zen, _, _, _ = erfa.atioq(cirs_ra, cirs_dec, astrom)
```

Comment:
Obviously this is totally in the noise for essentially any real scenario, but...
```
>>> timeit a, b = f()[:2]
10000000 loops, best of 3: 177 ns per loop
>>> timeit a, b = f()[:2]
10000000 loops, best of 3: 177 ns per loop
>>> timeit a, b, _, _, _ = f()
10000000 loops, best of 3: 99.8 ns per loop
>>> timeit a, b, _, _, _ = f()
10000000 loops, best of 3: 105 ns per loop
```

So apparently the current case *is* actually faster (first time I checked this...)